actioncable/lib/action_cable/test_helper.rb

```diff
@@ -0,0 +1,124 @@
+module ActionCable
+  # Provides helper methods for testing Action Cable broadcasting
+  module TestHelper
```

Comment:
Can we modify this to create an `ActionCable::TestCase`, similar to how Action Mailer has `ActionMailer::TestCase`, Active Job `ActiveJob::TestCase`, etc.?